backend/file_io.py

```diff
@@ -39,7 +39,7 @@ def log(*msg):
         tstamp = datetime.datetime.now()
         try:
             logfile = None
-            logfile = open('logfile.txt',"a")
+            logfile = open('otone_data/logfile.txt',"a")
```

Comment:
I'm planning on having all this file FileIO stuff removed and if anything we can use python's build in logger.